README.md

```diff
@@ -1,3 +1,74 @@
-# Backend app
+# Music Library
 
-This app was created with `@command-shift/create-backend-app`
+This repository contains an app that I built as a project for Command Shift bootcamp. It connects to a database and allows the user to create, read, update and delete data relating to artists and albums.
+
+This app was created using the NPX script `@command-shift/create-backend-app`.
+
+## Getting started
+
+### Set up the database
+
+- Install Docker.
+- Pull and run a postgres image to create a container.
+- Install pgAdmin.
+- Add a new server.
+
+### Clone this repository
+
+- Create a fork of this repo.
+- Copy the fork's git address and clone to you machine using `git clone`.
+
+### Install the project dependencies
+
+Use `npm install` to download the project's dependencies.
+
```

Comment:
You would also need to tell the user to add .env and .env.test files. Good way is to add a .env.example file to the GitHub repo so users can see what sort of thing they need to do